plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/LNode.java

```diff
@@ -80,6 +80,23 @@ public String getColor() {
             }
         }
 
+        public boolean isNorthSouthDummy() {
+            switch (this) {
```

Comment:
Even if there may be further cases in the future, I would rather write this `return this == NORTH_SOUTH_PORT`.

Reply:
Fine with me, just saw the PortConstraints do this the same way.